docs/installation/README.md

```diff
@@ -1,6 +1,21 @@
-# Installing Loki
+# Install Loki
 
-1. [Installing using Tanka (recommended)](./tanka.md)
-2. [Installing through Helm](./helm.md)
-3. [Installing through Docker or Docker Compose](./docker.md)
-4. [Installing locally](./local.md)
+## Installation methods
+
+Instructions for different methods of installing Loki and Promtail.
+
+- [Install using Tanka (recommended)](./tanka.md)
+- [Install through Helm](./helm.md)
+- [Install through Docker or Docker Compose](./docker.md)
+- [Install and run locally](./local.md)
+- [Install locally for manual build](./install-for-manual-build.md)
```

Comment:
```suggestion
- [Install from source](./install-for-manual-build.md)
```

Reply:
Could you also rename the new file and related sections from `install-for-manual-build` to `install-from-source` or something like this.

I think the phrase 'Install from source' is generally more recognizable.